build/steps-build-android.yml

```diff
@@ -23,6 +23,16 @@ steps:
 
 - template: templates/gitversion.yml
 
+- task: PowerShell@2
```

Comment:
should this only apply for android builds?

Reply:
you're right!